src/Components/Button/CircleButton.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen, fireEvent} from '@testing-library/react';
+import CircleButton from './CircleButton';
+
+// Test for Button Component Render
```

Comment:
I don't think these comments are necessary as the test() line includes a description of the test anyway